txgithub/scripts/create_token.py

```diff
@@ -21,25 +25,35 @@ def opt_scope(self, scope):
     def parseArgs(self, username):
         self['username'] = username
 
-def createToken(reactor, username, password, note, url, scopes):
-    from txgithub import token
-    d = token.createToken(username, password,
-            note=note, note_url=url,
-            scopes = scopes
-            )
-    d.addCallback(print)
+
+def createToken(reactor, username, password, note, url, scopes,
+                _createToken=token.createToken,
+                _print=print):
```

Comment:
This doesn't seem like a good interface, particular considering that underscores don't make arguments private.
On the other hand, does it make sense for any of the script stuff to be part of the public API?

In any case, I think it might make sense to split this PR into two, one adding coverage for the API, and one for the script coverage.